application/store/request.js

```diff
@@ -47,6 +47,10 @@ module.exports = Fluxxor.createStore({
     {
         var endpoint = this.state.endpoint[requestInfo.endpointName];
 
+        if (_.isUndefined(endpoint)) {
+            endpoint = this.state.endpoint[requestInfo.endpointName] = {};
```

Comment:
You should never set state directly, only through `this.setState`. `this.state` and `this.props` are both meant to be immutable.